services/order.py

```diff
@@ -0,0 +1,43 @@
+from django.db import transaction
+from django.db.models import QuerySet
+from db.models import Ticket, Order, User, MovieSession
```

Comment:
According to the checklist, you should use `get_user_model()` instead of directly importing and using `User`. This is considered best practice.